siliconcompiler/tools/openroad/rcx_bench.py

```diff
@@ -0,0 +1,94 @@
+from siliconcompiler.tools.openroad import openroad
+
+
+def setup_tool(chip):
+    ''' Helper method for configs specific to extraction tasks.
+    '''
+
+    openroad.setup(chip)
+
+    step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+    tool, task = chip._get_tool_task(step, index)
+
+    chip.set('tool', tool, 'task', task, 'script', 'sc_rcx.tcl', step=step, index=index)
+    chip.set('tool', tool, 'task', task, 'threads', 1, step=step, index=index)
```

Comment:
Just because we were talking about the `threads` in the meeting today. Does this actually do anything here?

Reply:
it will only open OpenROAD with a single thread, this task doesn't support multi threading